src/components/inputs/Selector.tsx

```diff
@@ -13,21 +13,46 @@ export enum SelectorType {
   STATUS_SELECTOR = 'statusSelector',
 }
 
+interface customOptions {
+  id: string
+  name: string
+  value?: string
+  //other custom types..
+}
+
 interface Prop {
   getSelectedValue: (value: unknown) => void
   startIcon: ReactNode
   value: unknown
   selectorType: SelectorType.STATUS_SELECTOR | SelectorType.ASSIGNEE_SELECTOR
   options: unknown[]
```

Comment:
can't we utilize TS `Generics` here rather that using `unknown`, please try exploring `Generics`